src/module/actors/drone-sheet.ts

```diff
@@ -0,0 +1,381 @@
+                          "swnr.drone.model.cuttlefish.text"
+                        ),
+                      },
+                    });
+                    break;
+                  case "ghostwalker":
+                    await this.actor.update({
+                      data: {
+                        health: { value: 1, max: 1 },
+                        baseAc: 15,
+                        model: "ghostwalker",
+                        fittings: 2,
+                        speed: 30,
+                        cost: 3000,
+                        encumbrance: 3,
+                        range: 5,
+                        tl: 4,
+                        description: game.i18n.localize(
+                          "swnr.drone.model.ghostwalker.text"
+                        ),
+                      },
+                    });
+                    break;
+                  case "sleeper":
+                    await this.actor.update({
+                      data: {
+                        health: { value: 8, max: 8 },
+                        baseAc: 12,
+                        model: "sleeper",
+                        fittings: 4,
+                        speed: 30,
+                        cost: 2500,
+                        encumbrance: 2,
+                        range: 100,
+                        tl: 4,
+                        description: game.i18n.localize(
+                          "swnr.drone.model.sleeper.text"
+                        ),
+                      },
+                    });
+                    break;
+                  case "pax":
+                    await this.actor.update({
+                      data: {
+                        health: { value: 20, max: 20 },
+                        baseAc: 16,
+                        model: "pax",
+                        fittings: 4,
+                        speed: 30,
+                        cost: 10000,
+                        encumbrance: 4,
+                        range: 100,
+                        tl: 5,
+                        description: game.i18n.localize(
+                          "swnr.drone.model.pax.text"
+                        ),
+                      },
+                    });
+                    break;
+                  case "alecto":
+                    await this.actor.update({
+                      data: {
+                        health: { value: 30, max: 30 },
+                        baseAc: 18,
+                        model: "alecto",
+                        fittings: 4,
+                        speed: 30,
+                        cost: 50000,
+                        encumbrance: 4,
+                        range: 5000,
+                        tl: 5,
+                        description: game.i18n.localize(
+                          "swnr.drone.model.alecto.text"
+                        ),
+                      },
+                    });
+                    break;
+                }
```

Comment:
This whole switch statement is a code smell. It contains data that should be totally external and blocks extension by users or other modules.

I've been tossing around the idea of a hull item for Ships which spec out the base details of a hull, though including an actor compendium for drones is probably a simpler method.

Reply:
Your idea for Ships sounds good to me. I added an item compendium pack for drone models. Drag and drop works and the drone sheet still contains a dropdown which is now populated by index for this pack.

One advantage of this approach is the process is symmetric for all the Actor types: we can universally use `Create New Actor` to make these entities and items to modify them.